lib/model/bloc/article/ArticleListProvider.dart

```diff
@@ -1,13 +1,13 @@
 import 'dart:async';
 
-import 'package:farmsmart_flutter/data/bloc/ViewModelProvider.dart';
-import 'package:farmsmart_flutter/data/bloc/article/ArticleDetailTransformer.dart';
-import 'package:farmsmart_flutter/data/bloc/article/ArticleListItemViewModelTransformer.dart';
-import 'package:farmsmart_flutter/data/model/article_entity.dart';
-import 'package:farmsmart_flutter/data/repositories/article/ArticleRepositoryInterface.dart';
-import 'package:farmsmart_flutter/model/loading_status.dart';
-import 'package:farmsmart_flutter/ui/discover/viewModel/ArticleListItemViewModel.dart';
-import 'package:farmsmart_flutter/ui/discover/viewModel/ArticleListViewModel.dart';
+import 'package:farmsmart_flutter/model/bloc/ViewModelProvider.dart';
+import 'package:farmsmart_flutter/model/bloc/article/ArticleDetailTransformer.dart';
+import 'package:farmsmart_flutter/model/bloc/article/ArticleListItemViewModelTransformer.dart';
+import 'package:farmsmart_flutter/model/model/article_entity.dart';
```

Comment:
/model/model ? 🤔

Reply:
🤔

Reply:
yes should be model/entities/. however I ha dot time box the changes to not take too long, maybe another spring clean PR a the eon of the week...